betterlockscreen

```diff
@@ -358,14 +359,14 @@ get_wall_list() {
     elif [ "${#paths[@]}" -eq 1 ]; then
         for ((i=0; i<${#DISPLAY_LIST[@]}; i++)); do
             # add same image to $WALL_LIST for each display
-            get_image "${paths[0]}"
+            get_image "${paths[0]}" "${DISPLAY_LIST[$i]}"
```

Comment:
We should just pass the resolution here, instead of parsing `${DISPLAY_LIST[$i]}` in `get_image()`.